### ISSUE TITLE
Open Source and Free Software

### ISSUE THREAD

**Issue by eniware-org** [ctubio/Krypto-trading-bot#1105]
What I am asking is a notice on the project page. One has to go much further inside to discover this.
Further, if you are providing something in exchange for payment you have to make sure to the best of your abilities it works. Many of us here are contributing to finding all kings of bugs that are not getting fixed. Some are critical and lead to financial loses - PING-PONG !
In addition, you are removing functionality from the gateway for which I have paid and need. I refer to BITFINEX_MARGIN here. I don't know how to call this!

_Originally posted by @eniware-org in https://github.com/ctubio/Krypto-trading-bot/issues/1104#issuecomment-843179313_

**Comment by ctubio** [comment]
as said on https://github.com/ctubio/Krypto-trading-bot/issues/1104#issuecomment-843111506 the notice is at the step 0 of the installation

it is also in the first line of the output of `--help`

it is also readable on the makefile

if still you dont like it, feel free to do an acceptable PR with the modifications that you consider best

additionally, margin was removed because was not working,a nd because was badly implemented, not because other reasons, so i dont understand how you make use of it (i mean expected proper use of it)

you should be happy to not run shit code, and be happy for it to be removed because it can be put it back properly

aand it was removed at https://github.com/ctubio/Krypto-trading-bot/commit/7a6a72f0932b2c713f8782d8541c8dfbd8fce0db, as you can see is a small commit; to put margin trading back on bitfinex i just need some free time since im alone here

if you wanna speed up this development you can do things that are not nice for me to suggest and that other ppl usualy do; but since you feel that you already contributed enough let me just ask you to stop duplicating your comments, im sure you can do that; and you can be sure that i understood that you paid and need for BITFINEX_MARGIN even if was not properly working; thats why it will be implemented next instead of some other exchange (together with binance margin).

err.. since all this is about BITFINEX_MARGIN but almost no lines are related, let me just close this

i know you want BITFINEX_MARGIN, but if you want the issue opened consider to open a BITFINEX_MARGIN  issue

**Comment by eniware-org** [comment]
Hi @ctubio . I started a discussion about BITFINEX_MARGIN in https://github.com/ctubio/Krypto-trading-bot/discussions/1083#discussion-3254673 about 2 months ago.
Willing to contribute with what is needed and already have views as to what is needed there.
You mentioned that you have something much more simpler in mind about margins and I left the issue with the missing release to rest. As nothing has progressed, I raised the issue with the missing release. By the way, there are much older releases there. Just not the one I need!
Anyway, no need to escalate bad feelings - they shorten life. Trying to be positive.
